composer.json

```diff
@@ -18,22 +18,11 @@
         "ext-simplexml": "*",
         "guzzle/guzzle": "^2.8|^3.0",
         "psr/log": "^1.0",
-        "symfony/config": "^2.0|^3.0",
+        "symfony/config": "^2.6|^3.0",
```

Comment:
Guys, would it be possible to reconsider the version of ``symfony/config`` and replace it with ``2.3``? There is now an issue with testing bundles for Symfony 2.3, which is still officially supported. Details here:
https://github.com/satooshi/php-coveralls/commit/e15c5e38cd3fb2f06f2ba22c3e25962ac7d0a5b4#commitcomment-15540997